src/components/ui-shell/ui-shell-preview.hbs

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Do you want to move this to `demo/views/layouts` so other components can use this preview potentially…?

Reply:
Maybe. Would rather wait to move it when the need is there than move it prematurely.